torch/special/__init__.py

```diff
@@ -230,6 +230,48 @@
     tensor([ 0.,  1.])
 """.format(**common_args))
 
+xlog1py = _add_docstr(_special.special_xlog1py,
+                      r"""
+xlog1py(input, other, *, out=None) -> Tensor
+
+Computes ``input * log1p(other)`` with the following cases.
+
+.. math::
+    \text{out}_{i} = \begin{cases}
+        \text{NaN} & \text{if } \text{other}_{i} = \text{NaN} \\
+        0 & \text{if } \text{input}_{i} = 0.0 \\
+        \text{input}_{i} * \text{log1p}(\text{other}_{i})& \text{otherwise}
+    \end{cases}
+
+Similar to SciPy's `scipy.special.xlog1py`.
+
+""" + r"""
+
+Args:
+    input (Number or Tensor)
+    other (Number or Tensor)
```

Comment:
![Screenshot from 2021-04-23 13-11-59](https://user-images.githubusercontent.com/19503980/115836806-8d8d4600-a435-11eb-846c-12317e42203a.png)

Should we update the following lines like above? Right now it looks like 
![Screenshot from 2021-04-19 11-34-05](https://user-images.githubusercontent.com/19503980/115836946-b01f5f00-a435-11eb-9d26-fad8da141f6e.png)


Suggested:
```
Args:
    input (Number or Tensor) : Multiplier
    other (Number or Tensor) : Argument
```

Reply:
I also noticed that (other ops have the same issue). Sure, your suggestion makes sense and would look nicer

Reply:
Updating it for `xlog1py` and `xlogy`. Thanks!